src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -837,6 +837,22 @@ type Geometry {
     points: Polyline
 }
 
+scalar GeoJson
+
+type StopGeometries {
+    """Representation of the stop geometries as GeoJSON (https://geojson.org/)"""
+    geoJson: GeoJson,
+
+    """
+    Representation of a stop as a series of polylines.
```

Comment:
Describe the station geometry.

Reply:
I opted to add the documentation on the StopGeometries type as it applies to both representations: https://github.com/ibi-group/OpenTripPlanner/commit/6b738ac42ea68dec3bf0e7491597a278cdfe67cb